docker_images/mbed-os-env/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# ------------------------------------------------------------------------------
+# Pull base image
+FROM ubuntu:20.04
+
+# ------------------------------------------------------------------------------
+# Arguments
+ARG WORKDIR=/root
+
+# ------------------------------------------------------------------------------
+# Install tools via apt
+RUN ls
+
+ADD requirements.txt .
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -x \
+    && apt -y update \
+    && apt -y install \
+    git \
+    wget \
+    python3 \
+    python3-dev \
+    python3-setuptools \
+    python3-usb \
+    python3-pip \
+    software-properties-common \
+    build-essential \
+    astyle \
+    mercurial \
+    ninja-build \
+    libssl-dev \
+    cargo \
```

Comment:
Why do we need rust ?

Reply:
good spot. I just copy pasted from existing docker file. I will need to look into each dependency carefully.